First attempt to add CIVET test results to RTM

aeslaughter opened the pull request:
(refs #13978)

<!--
INCLUDE THE FOLLOWING IN THE PR DESCRIPTION
- Explain relevant design information for your change.
- Follow the [Coding Standards](http://mooseframework.org/wiki/CodeStandards/).
- Submit or improve [Test Cases](http://mooseframework.org/wiki/MooseTraining/testing/).
- Reference a specific issue, place "refs #<issue>" or "closes #<issue>" (e.g., #closes #1234).
-->

aeslaughter committed:
Adds CIVET test results to RTM

(refs #13978)

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/446133/) on 478da27 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/14537/site)

This comment will be updated on new commits.

aeslaughter committed:
Update tests for changes to MooseDocs

(refs #13978)

aeslaughter committed:
Improve API for civet extension and allow download directory to be modified

aeslaughter committed:
Allow CIVET extension to be optional and limit testing in latex

aeslaughter committed:
Small changes for test machines

- Add Python2 support to meet min. target
- Set the cache location for civet extension

(refs #13978)

aeslaughter commented:
@permcody 
[RTM](https://mooseframework.inl.gov/docs/PRs/14537/site/sqa/framework_rtm.html)
[Summary](https://mooseframework.inl.gov/docs/PRs/14537/site/civet/)

permcody commented:
Looks pretty good!

On Mon, Dec 23, 2019 at 8:50 PM Andrew E Slaughter <notifications@github.com>
wrote:

> @aeslaughter <https://github.com/aeslaughter> requested your review on:
> #14537 <https://github.com/idaholab/moose/pull/14537> First attempt to
> add CIVET test results to RTM.
>
> —
> You are receiving this because your review was requested.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/14537?email_source=notifications&email_token=AAXFOIES3GNDWTIXUPGML5TQ2GBJ3A5CNFSM4J4DQEMKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOVU3T2YA#event-2906078560>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXFOIAKRCMKA6JCEF2OUULQ2GBJ3ANCNFSM4J4DQEMA>
> .
>
